### PR TITLE
Fix sonar codesmells #555

### PR DESCRIPTION
**Pull request by YuriiDubnytskyi**
Fix sonar code smells
 - moved nested ternary operators into separate functions and files (utils directory)
 - refactor functions for Cognitive Complexity (create separate functions and files)

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/chat/active-messenger/active-messenger.js [hunk 1]
```diff
@@ -65,9 +66,7 @@ export const ActiveMessenger = ({ themeMode, visible, mailFormVisible }) => {
     setOpen(false);
   };
 
-  const Alert = (props) => (
-    <MuiAlert elevation={6} variant='filled' {...props} />
-  );
+  const Alert = (props) => <MuiAlert elevation={6} variant='filled' {...props} />;
```
Comment (yuliiakucher) [inline src/containers/chat/active-messenger/active-messenger.js:69]: please remove magic string

File: src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js [hunk 2]
```diff
@@ -50,7 +46,7 @@ const SearchBarListItem = ({ product }) => {
             innerText={<ShoppingBasketIcon />}
             onClickHandler={() => {}}
             buttonStyle='classic'
-          />*/}
```
Comment (yuliiakucher) [inline src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js:53]: please remove commented code

**Review by yuliiakucher — CHANGES_REQUESTED** [review]
please fix comment

File: src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js (hunk 2 above)
Comment (YuriiDubnytskyi) [reply src/containers/search-bar-list/search-bar-list-item/search-bar-list-item.js:53]: fixed

**Review by grebiniuk — APPROVED** [review]

**Review by yuliiakucher — APPROVED** [review]

**Pull request merged**